packages/cli/src/lib/decorators/preconditions/ng.ts

```diff
@@ -0,0 +1,43 @@
+import type Command from '@oclif/command';
+import {cli} from 'cli-ux';
+import dedent from 'ts-dedent';
+import {getPackageVersion} from '../../utils/misc';
+import {appendCmdIfWindows} from '../../utils/os';
+import {spawnProcess} from '../../utils/process';
+import {getBinInstalledPrecondition} from './binPreconditionsFactory';
+
+const angularCliPackage = '@angular/cli';
+async function installAngularCli(target: Command): Promise<boolean> {
+  const shouldInstall = await cli.confirm(
+    dedent`
+        You need to have the Angular-CLI installed on your computer before proceeding.
+        Do you want to install it now? (y/n)`
+  );
+  if (!shouldInstall) {
+    return false;
+  }
+  const version = getPackageVersion(angularCliPackage);
+  target.log(`Running npm install --global @angular/cli@${version}.`);
+  const output = await spawnProcess(appendCmdIfWindows`npm`, [
```

Comment:
It's starting to get a bit messy here, since it means we have a precondition that depends on one another (that npm is installed and new enough that you can actually install the angular cli with it).

Could we just print an info message about how to install it ?

It should be pretty easy for them to do it anyway, plus they have more control on the installation process, I guess...

Do you think there's a lot of value by us trying to do the installation "automatically" ?

Reply:
Honestly, not much value. I tried to match what was possible in `vue` and `react` but that's all. I do not mind axing the auto-update tbh

Reply:
- [ ] Remove autofix